static/docs/install/completion.md

```diff
@@ -27,15 +27,24 @@ Depending on what you typed on the command line so far, it completes:
 - Arguments for selected options. For example, `dvc repro` completes with stage
   files to reproduce.
 
-## Which shell you have
+## What shell do you have?
 
 Use the command `echo $0` to check your active shell, if you are not sure which
 one you are using (Bash or Zsh).
 
 Depending on the shell that you are using, follow the instruction below to
 install and configure shell completion.
 
-## Bash completion on Mac OS
+## Manual setup
+
+Note that Shell completion is automatically included when DVC is installed on
```

Comment:
Again, I would simplify and put it as a first sentence into each section below. And would keep H2 (for better navigation)

Reply:
This doc is specifically about completion, so maybe we can keep the details here? Or are you thinking in the future more installation methods will include shell completion so these comments may not be needed?

Reply:
yeah, but having something like `Install on Mac Os`, `Install on Linux`, etc seems to be more relevant/actionable than just a single `Manual Setup` item on the right side menu - that was my thinking.

Reply:
Ah yes well that's an engine thing. I didn't think to check the side bar, which only cares about H2, it seems.. I'm changing the headers again then... Please check! https://dvc-org-pr-725.herokuapp.com/doc/install/completion